engine/tests/ElggCoreGetEntitiesFromAnnotationsTest.php

```diff
@@ -330,13 +330,17 @@ public function testElggGetEntitiesFromAnnotationsCalculateConstrainedByWhere()
 			}
 		}
 
+		$prefix = _elgg_services()->db->getTablePrefix();
 		$options = array(
 			'type' => 'object',
 			'subtypes' => $subtypes,
 			'limit' => $num_entities,
 			'annotation_name' => $name,
 			'annotation_values' => array_unique(call_user_func_array('array_merge', $values)),
 			'calculation' => 'sum',
+			'joins' => array(
+				"JOIN {$prefix}metastrings msv ON (n_table.value_id = msv.id)",
```

Comment:
the commit i added is here: https://github.com/propertunist/Elgg/commit/a6590a9a68f534b8ae6bde274bf642bec301f4a3